podcastScreen/podcast_screen/lib/main.dart

```diff
@@ -54,107 +54,96 @@ class _PodcastPageState extends State<PodcastPage> {
           ),
           SliverToBoxAdapter(
             child: Container(
+              padding: const EdgeInsets.all(16.0),
               child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
                 children: <Widget>[
                   _buildPodcastCard(),
-                  SizedBox(
-                    height: 8.0,
-                  ),
                   _buildPodcastCard(),
-                  SizedBox(
-                    height: 8.0,
-                  ),
                   _buildPodcastCard(),
-                  SizedBox(
-                    height: 8.0,
-                  ),
                   _buildPodcastCard(),
-                  SizedBox(
-                    height: 8.0,
-                  ),
                   _buildPodcastCard(),
-                  SizedBox(
-                    height: 8.0,
-                  ),
                   _buildPodcastCard(),
-                  SizedBox(
-                    height: 8.0,
-                  ),
                   _buildPodcastCard(),
-                  SizedBox(
-                    height: 8.0,
-                  ),
                 ],
               ),
             ),
           )
         ],
       ),
+      floatingActionButton: FloatingActionButton.extended(
```

Comment:
Extended FAB is used to show the primary action of the screen. It allows the user to resume the playback of the podcast.